src/Sparrow.Server/ByteString.cs

```diff
@@ -671,6 +671,7 @@ public sealed class ByteStringContext : ByteStringContext<ByteStringMemoryCache>
         public const int MaxAllocationBlockSizeInBytes = 256 * MinBlockSizeInBytes;
         public const int DefaultAllocationBlockSizeInBytes = 1 * MinBlockSizeInBytes;
         public const int MinReusableBlockSizeInBytes = 8;
+        public const int MaxAllocatedBlockSize = 2 * Sparrow.Global.Constants.Size.Megabyte;
```

Comment:
I would call that `SegmentDefaultSizeInBytes` or something like that, because it is not allocated yet.

Reply:
This is actually the max size in bytes.

Reply:
changed